src/dirtyfields/compat.py

```diff
@@ -13,6 +14,11 @@ def is_db_expression(value):
         return isinstance(value, (BaseExpression, Combinable))
 
 
+def is_deferred(model, field):
```

Comment:
Usually the `compat` module is used to contain functions that wraps different behaviours depending on Django versions.

I will move this one somewhere else, as it's  not related to Django version compatibility.